src/src/pages/capabilities/resources/resourceInfoBadges/index.js

```diff
@@ -186,6 +228,77 @@ export function ResourceInfoBadges() {
           </div>
         </>
       )}
+
+      <hr className={styles.divider} />
+
+      <p style={{ textAlign: "center" }}>
+        <img src={azureLogo} alt="Azure icon" style={{ height: "2.5rem" }} />
+      </p>
+
+      <div className={styles.azure}>
+        <div className={styles.items}>
+          {azureResourcesList != [] && isLoadedAzure ? (
+            azureResourcesList.map((x) => (
+              <div key={x.id}>
+                <div className={styles.environment}>
+                  Azure resources for the {x.environment} environment:
+                </div>
+                <div className={styles.azureresource}>
+                  <Badge>
+                    <strong>{x.id} </strong>
+                  </Badge>
+                </div>
+              </div>
+            ))
+          ) : (
+            <div>
+              {" "}
+              <strong>
+                <em style={{ color: theme.colors.text.secondary.primary }}>
+                  No Azure resources linked with this Capability.
```

Comment:
I am perfectly okay with this decision.

However, did we consider if we should show a spinner or some other loading indicator instead?
This message might be confusing to users who know they have resources, and the resources simply haven't loaded yet.

The load times might be so small, that this is merely of theoretical concern, so feel free to ignore this :)